app/ios/iosimagepicker.cpp

```diff
@@ -36,6 +36,16 @@ void IOSImagePicker::showImagePicker( int sourceType, const QString  &targetDir
 #endif
 }
 
+QString IOSImagePicker::readExif( const QString &filepath, const QString &tag )
+{
+#ifdef Q_OS_IOS
+  return readExifDirect( filepath, tag );
+#else
+  Q_UNUSED( filepath )
+  Q_UNUSED( tag )
+#endif
+}
+
 QString IOSImagePicker::targetDir() const
```

Comment:
why this function is changed?

Reply:
There is now extra handling for gallery image - the image file is copied during the imagePicker action.
For camera image, its kept as it was - image data are sent as response data and saved in this callback.